bin/MadGraph5_aMCatNLO/cards/production/13TeV/HPlusHF/HPlusBottom_5FS_MuRFScaleDynX0p50_M125_TuneCP5_13TeV_amcatnloFXFX_pythia8/HPlusBottom_5FS_MuRFScaleDynX0p50_M125_TuneCP5_13TeV_amcatnloFXFX_pythia8_param_card.dat

```diff
@@ -0,0 +1,96 @@
+######################################################################
+## PARAM_CARD AUTOMATICALY GENERATED BY MG5 FOLLOWING UFO MODEL   ####
+######################################################################
+##                                                                  ##
+##  Width set on Auto will be computed following the information    ##
+##        present in the decay.py files of the model.               ##
+##        See  arXiv:1402.1178 for more details.                    ##
+##                                                                  ##
+######################################################################
+
+###################################
+## INFORMATION FOR LOOP
+###################################
+Block loop 
+    1 9.118800e+01 # MU_R 
+
+###################################
+## INFORMATION FOR MASS
+###################################
+Block mass 
+    4 0.000000e+00 # MC 
+    5 0.000000e+00 # MB 
+    6 1.730000e+02 # MT 
+   15 1.777000e+00 # MTA 
+   23 9.118800e+01 # MZ 
+   25 1.250000e+02 # MH 
+## Dependent parameters, given by model restrictions.
+## Those values should be edited following the 
+## analytical expression. MG5 ignores those values 
+## but they are important for interfacing the output of MG5
+## to external program such as Pythia.
+  1 0.000000e+00 # d : 0.0 
+  2 0.000000e+00 # u : 0.0 
+  3 0.000000e+00 # s : 0.0 
+  11 0.000000e+00 # e- : 0.0 
+  12 0.000000e+00 # ve : 0.0 
+  13 0.000000e+00 # mu- : 0.0 
+  14 0.000000e+00 # vm : 0.0 
+  16 0.000000e+00 # vt : 0.0 
+  21 0.000000e+00 # g : 0.0 
+  22 0.000000e+00 # a : 0.0 
+  24 8.041900e+01 # w+ : cmath.sqrt(MZ__exp__2/2. + cmath.sqrt(MZ__exp__4/4. - (aEW*cmath.pi*MZ__exp__2)/(Gf*sqrt__2))) 
+
+###################################
+## INFORMATION FOR SMINPUTS
+###################################
+Block sminputs 
+    1 1.325070e+02 # aEWM1 
+    2 1.166390e-05 # Gf 
+    3 1.180000e-01 # aS 
+
+###################################
+## INFORMATION FOR YUKAWA
+###################################
+Block yukawa 
+    4 0.000000e+00 # ymc 
+    5 4.180000e+00 # ymb 
+    6 0.000000e+00 # ymt 
```

Comment:
if you use 5f scheme, why ymb is not zeto, but ymt and ymtau are zero

Reply:
the masses and yukawa couplings can be treated as independent values in these simulations, which are at QCD NLO (I think this was also mentioned in a recent comment here: [cms-sw/genproductions#3135 (comment)](https://github.com/cms-sw/genproductions/pull/3135#issuecomment-1092821296)). What should be consistent is the flavour scheme and the value of the masses (not yukawa), so for 5FS we set MB=0 and MC=0, for 4FS MB!=0 MC=0, and for 3FS MC!=MB!=0`.